src/Swarrot/Processor/SignalHandler/SignalHandlerProcessor.php

```diff
@@ -51,6 +55,10 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function sleep(array $options)
     {
+        if (!extension_loaded('pcntl')) {
+            return false;
```

Comment:
this should return ``true``. Otherwise, you always stop it when the queue is empty (which is the job of the InsomniacProcessor)